File: qualifier-week-4/graded-assignments/GrPA7 - Graded.c
#include <stdio.h>

int main() {
    int n,  total_cookies = 0, daily_sales;

    scanf("%d", &n);

    if (n <= 0) {
        return 1;
    }

    for (int i = 0; i < n; ++i) {
        scanf("%d", &daily_sales);
        total_cookies += daily_sales;
    }
    printf("%d", total_cookies);
    return 0;
}

File: qualifier-week-4/graded-assignments/GrPA7 - Graded.rs
use std::io::{self, Read};

fn main() {
    let mut num_days_str = String::new();
    io::stdin().read_to_string(&mut num_days_str).expect("Input Error: Failure");

    let num_days: u32 = match num_days_str.trim().parse() {
        Ok(days) => days,
        Err(_) => {
            return;
        }
    };

    if num_days <= 0 {
        return;
    }

    let mut total_cookies = 0;
    for day in 1..(num_days + 1) {
        let mut daily_sales_str = String::new();
        io::stdin().read_to_string(&mut daily_sales_str).expect("Input Error: Failure");

        let daily_sales: u32 = match daily_sales_str.trim().parse() {
            Ok(sales) => sales,
            Err(_) => {
                return;
            }
        };

        total_cookies += daily_sales;
    }

    println!(total_cookies);
}
